prime-router/src/test/kotlin/validation/MessageValidatorTests.kt

```diff
@@ -15,7 +15,7 @@ import org.junit.jupiter.api.Nested
 import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.assertThrows
 
-private const val radxmarxProfileLocation = "metadata/hl7_validation/v251/radxmars"
+private const val radxmarxProfileLocation = "metadata/hl7_validation/v251/radxmars/production"
```

Comment:
If this is a typo I don't want it to be fixed because it made me chuckle. The variable name here is radxmar**x**. Sounds like "radical Marx" 😂

Reply:
lol I think it was an existing typo... I didn't even notice